### PR TITLE
Supervisor visual changes

### LINKED ISSUE #4758: Modify visualization on view all supervisors page
**What type(s) of user does this feature affect?**
 - supervisors?
 - admins?

**Description**
On the `/supervisors` page the red + green + gray bubbles do  not add up to the total number of volunteers. While the legend at the top indicates that the red + green should add up to the total number of volunteers, and the gray bubble is just a count of transition-aged youth, this is still visually confusing for the stakeholders. Modify this in some way so that it doesn't seem like the bubbles should add up to the total number of volunteers

**Screenshots of current behavior, if any**  
Screen Shot 2023-04-14 at 11 29 35 AM

_Login Details:_  
[Link to QA site](https://casa-qa.herokuapp.com/)  
[Link to new theme site](https://casa-new-theme.herokuapp.com/languages/new)  

Login Emails: 
- volunteer1@example.com  view site as a volunteer
- supervisor1@example.com view site as a supervisor
- casa_admin1@example.com view site as an admin
- all_casa_admin1@example.com view site as an all casa admin
  - go to `/all_casa_admins/sign_in`  

password for all users: 12345678  

### Questions? Join Slack!

We highly recommend that you join us in slack https://rubyforgood.herokuapp.com/ #casa channel to ask questions quickly and hear about office hours (currently Tuesday 5-7pm Pacific), stakeholder news, and upcoming new issues.

### PR DESCRIPTION
**Pull request by rae-stanton**
This adjusts the tests to be reflective of their now relevant Bootstrap classes versus the CSS from before. I'm not sure we needed the custom CSS since Bootstrap gives it to us for free, but I am happy to either keep the CSS file or we can remove it and the changes won't be affected from what I can see. 

### What github issue is this PR for, if any?
Resolves #4758 

### What changed, and why?
This refactored the previous visual where all of the classes were inline with each other, versus being on corresponding rows that made a little more sense to have separated based on volunteers and transitioned youth. I also changed the spans to divs. This should also keep everything within a flexbox that doesn't go past the "actions" column and beyond. The second flex row should also stay within a 50% margin overall just for visual sake. 

### How will this affect user permissions?
No permissions were changed, so I guess:
- Supervisor permissions: visual changes
- Admin permissions: visual changes

### How is this tested? (please write tests!) 💖💪
Tests were adjusted to reflect use of bootstrap 5 classes versus the custom CSS from before.

### Screenshots please :)
Screenshot 2023-04-29 at 1 52 56 PM

### CODE REVIEW

**Review by littleforest — COMMENTED** [review]
Thanks for working on this! I'm all for removing the CSS file if it is no longer needed. Can you take a look at the erblint error and see what fixes need to be made?

**Comment by rae-stanton** [comment]
@littleforest Absolutely! I think it may be a config issue on my end, I'll update in the AM. 😊

**Comment by rae-stanton** [comment]
@littleforest I totally added extra commas to separate lines when I didn't need to - total oops on my part, but everything should be good to go now! 😊

**Comment by rae-stanton** [comment]
@littleforest Ok this should now all be ready to go! Let me know if you see any additional changes that need to be made. 😊

File: spec/views/supervisors/index.html.erb_spec.rb [hunk 1]
```diff
@@ -35,29 +35,29 @@
...
+      #   active_flex = active_bar.inner_html
+      #   inactive_flex = inactive_bar.inner_html
+      #   active_content = active_bar.children[0].text.strip
+      #   inactive_content = inactive_bar.children[0].text.strip
+
+      #   expect(active_flex).to eq(active_content)
+      #   expect(inactive_flex).to eq(inactive_content)
+      #   expect(active_flex.to_i).to eq(2)
+      #   expect(inactive_flex.to_i).to eq(1)
+      # end
```
Comment (littleforest) [inline spec/views/supervisors/index.html.erb_spec.rb:60]: @rae-stanton is this test broken now? Can it be modified to match the new layout?

Comment (rae-stanton) [reply spec/views/supervisors/index.html.erb_spec.rb]: @littleforest oops! I thought I committed the final test - it's passing now and I've added the updated commit for the test file.

**Review by littleforest — COMMENTED** [review]
Could you just run `standardrb --fix` to fix the linting? The spec failure is a known issue and not a problem.

**Review by littleforest — APPROVED** [review]
Thanks for adding tests!

**Pull request merged**